### PR TITLE
pdctl: support filter json output by calling external jq

### PR DESCRIPTION
**Pull request by disksing**
## What have you changed? (required)
- support  jq query to format the output of _region_ and _store_ command
- import package `shellwords` to split command line strings
- Note: it requires `jq` be installed in the system. We may want to include a `jq` binary in our release package.

## What are the type of the changes (required)?
- New feature (non-breaking change which adds functionality)

## How has this PR been tested (required)?
- start a local cluster to test jq queries.

## Does this PR affect documentation (docs/docs-cn) update? (optional)
Yes. See https://github.com/pingcap/docs-cn/pull/790

### CODE REVIEW

File: pdctl/command/region_command.go [hunk 1]
```diff
@@ -42,7 +43,7 @@ type regionInfo struct {
 // NewRegionCommand return a region subcommand of rootCmd
 func NewRegionCommand() *cobra.Command {
 	r := &cobra.Command{
-		Use:   "region <region_id>",
+		Use:   `region <region_id> [-jq="<query string>"]`,
```
Comment (siddontang) [inline pdctl/command/region_command.go]: only support some commands?

File: pdctl/command/region_command.go [hunk 2]
```diff
@@ -82,6 +84,11 @@ func showRegionCommandFunc(cmd *cobra.Command, args []string) {
 		fmt.Printf("Failed to get region: %s\n", err)
 		return
 	}
+	if flag := cmd.Flag("jq"); flag != nil && flag.Value.String() != "" {
```
Comment (siddontang) [inline pdctl/command/region_command.go]: can we provide a Print interface for jq or standard output?

File: pdctl/command/region_command.go (hunk 1 above)
Comment (disksing) [reply pdctl/command/region_command.go]: Yes. Only support APIs that query region/store list. The results returned by other APIs are simple and easy to see with the naked eye.

File: pdctl/command/region_command.go (hunk 2 above)
Comment (disksing) [reply pdctl/command/region_command.go]: It's possible but I don't think it's necessary.

Comment (siddontang) [reply pdctl/command/region_command.go]: seem better to extract to a function like `isJQUsed`

Comment (disksing) [reply pdctl/command/region_command.go]: There is no need to over-abstract, `--jq` is not a general parameter. these two commands just happen to both support it.

**Pull request merged**